qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -135,7 +135,9 @@ def getNodeItemForNode(self, node, highlight_level):
         if name is None:
             print("Error, no label defined for node with attr: %s" % node.attr)
             return None
-        name = name.decode('string_escape')
+
+        import codecs # To work with Python 2 and 3 (https://stackoverflow.com/a/23151714)
```

Comment:
Can you please move the import to the top of the file (between line 32 and 33) and remove the comment.